pkg/get/tools.go

```diff
@@ -1458,5 +1458,53 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 		},
 	)
 
+	tools = append(tools,
+		Tool{
+			Owner:       "sigstore",
+			Repo:        "cosign",
+			Name:        "cosign",
+			Version:     "0.6.0",
+			Description: "Container Signing, Verification and Storage in an OCI registry.",
+			BinaryTemplate: `
```

Comment:
Do we need BinaryTemplate and UrlTemplate?

Reply:
without it, it will be downloaded but not copied in the arkade folder

its like this in the /tmp folder.
![image](https://user-images.githubusercontent.com/38325136/126880394-f8899233-0cb6-42fd-8ca6-5afd5dd3a654.png)

only the binary template picked it up correctly.

Working only with binary is not working either, as they have a version in der filename and i run into the v<-String

Reply:
Its always a challenge, when the binary looks different, somtimes as tar.gz, somtimes .gz then with version in the name somtimes without. I love this challanges 😍

Reply:
no, i checked again. its because the linux binary is corrupted i thougt it did not copied it..